Snippets/ssf2.sqlpromptsnippet

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I'm not sure this makes sense to add for 2 reasons —

1) doesn't it block use of `ssf`?
2) why bother having a version of a snippet that deals with formatting; when SQL Prompt does auto-formatting anyway?

I'd advocate removing :)